Helpers.gs

```diff
@@ -833,26 +833,29 @@ function sendSummary() {
  */
 function callWithBackoff(func, maxRetries) {
   var tries = 0;
-  var timeout = 100;
   var result;
-  while(tries < maxRetries){
+  while(true){
```

Comment:
Let's keep the condition here. In case something unintended/unexpected happens with the below code I'd rather it still hit this condition rather than become an infinite loop

Reply:
Yes, makes sense.